### PR TITLE
Added a base ship class that can move around. Also, a base player class that extends ship and handles the user inputs.

### PR DESCRIPTION
**Pull request by LewisRhine**

### CODE REVIEW

File: classes/player.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import pygame
+
+from classes.ship import Ship
+
+
+class Player(Ship):
+
+    def __init__(self, x: int, y: int):
+        self.image = pygame.image.load('assets/ships/Ship2/Ship2.png').convert_alpha()
+        self.image = pygame.transform.rotate(self.image, 90)
```
Comment (JonR205) [inline classes/player.py:12]: What is this for?

File: classes/player.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        super().__init__(x, y, 1, 15)
+
+    def player_input(self):
+        keys = pygame.key.get_pressed()
+        if keys[pygame.K_a]: self.turn(-self.engines / 2)
+        if keys[pygame.K_d]: self.turn(self.engines / 2)
+        if keys[pygame.K_w]:
+            self.accelerate(5)
```
Comment (JonR205) [inline classes/player.py]: Why 5 for accelerate

File: classes/player.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+class Player(Ship):
+
+    def __init__(self, x: int, y: int):
+        self.image = pygame.image.load('assets/ships/Ship2/Ship2.png').convert_alpha()
+        self.image = pygame.transform.rotate(self.image, 90)
+        super().__init__(x, y, 1, 15)
+
+    def player_input(self):
+        keys = pygame.key.get_pressed()
+        if keys[pygame.K_a]: self.turn(-self.engines / 2)
```
Comment (JonR205) [inline classes/player.py]: Not sure how this works

File: classes/player.py (hunk 1 above)
Comment (LewisRhine) [reply classes/player.py:12]: Added explanation

**Review by JonR205 — APPROVED** [review]

**Pull request merged**